Sources/Core/Microsoft.StreamProcessing.Provider/IGroupedWindow.cs

```diff
@@ -0,0 +1,25 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+namespace Microsoft.StreamProcessing.Provider
+{
+    /// <summary>
+    ///
+    /// </summary>
+    /// <typeparam name="TKey"></typeparam>
+    /// <typeparam name="TPayload"></typeparam>
```

Comment:
Can we fill these out? Probably best to not add more empty documentation if avoidable :)